docs/devenv_rhel8.md

```diff
@@ -78,6 +80,7 @@ The artifact of the build is the `microshift` executable file located in the `_o
 ### RPM Packages
 Run make command with the `rpm` or `srpm` argument in the top-level directory. 
 ```bash
+sed -i 's,^BuildRequires: golang,# &,' ./packaging/rpm/microshift.spec
```

Comment:
I would suggest permanently commenting out this requirement until we get the right RPM package. 
Can we do this in the .spec and leave an appropriate comment?

Reply:
But this works in CI - image used is based on rhel8 with go 1.18. This change is only needed for local dev env

Reply:
I know, but still it requires unmanaged edits of source files, which is messy. 
If we do this, we would need to explain why we're doing this. I think we could avoid all the mess by temporary commenting out this statement if it does not have other consequences.

Reply:
Done